docs/docs/graph-recipe.mdx

```diff
@@ -0,0 +1,128 @@
+---
+id: graph-recipe
+sidebar_label: Graph Recipe
+title: Graph Recipe
+description: Learn about Graph Recipe for Rasa Open Source.
+abstract: Graph recipes provide a more fine tuned configuration for your executable graphs.
+---
+
+We now support graph recipes in addition to the default recipe. What graph recipes provide is more powerful control over how execution graph schemas are built.
+
+:::caution New in 3.1
+This feature is experimental.
+We introduce experimental features to get feedback from our community, so we encourage you to try it out!
+However, the functionality might be changed or removed in the future.
+If you have feedback (positive or negative) please share it with us on the [Rasa Forum](https://forum.rasa.com).
+
+:::
+
+
+## Differences with Default Recipe
+
+There are some differences between the default recipe and the new graph recipe. Main differences are:
+
+- Default recipe is named `default.v1` in the config file whereas graph recipes are named `graph.v1`.
+- Default recipes provide an easy to use recipe structure whereas graph recipes are more advanced and powerful.
+- Default recipes are very opinionated and provide various defaults whereas graph recipes are more explicit.
+- Default recipes can auto-configure themselves and dump the defaults used to the file if some sections in `config.yml` are missing, whereas graph recipes do none of this and assume what you see is what you get. There are no surprises with graph recipes.
+- Default recipe divides graph configuration into mainly two parts: `pipeline` and `policies`. These can also be described as NLU and core (dialogue management) parts. For graph recipe on the other hand, the separation is between training (ie. `train_schema`) and prediction (ie. `predict_schema`).
+
+:::tip Starting from scratch?
+
+If you don't know which recipe to choose, use the default recipe to bootstrap your project fast. If later you find that you need more powerful finetuning options, you can always change your recipe to be a graph recipe.
```

Comment:
"finetuning" has a double meaning here 😁

Reply:
Double speak? How? :)

Reply:
We use it to mean incremental training

Reply:
Oh yes, I see what you mean. I can change "finetuning" to "configuration" here?